clients/runtime/src/rpc.rs

```diff
@@ -1,425 +1,410 @@
 use crate::{
-	conn::{new_websocket_client, new_websocket_client_with_retry},
-	metadata,
-	metadata::{DispatchError, Event as SpacewalkEvent},
-	notify_retry,
-	types::*,
-	AccountId, Error, RetryPolicy, SpacewalkRuntime, SpacewalkSigner, SubxtError,
+    conn::{new_websocket_client, new_websocket_client_with_retry},
+    metadata,
+    metadata::{DispatchError, Event as SpacewalkEvent},
+    notify_retry,
+    types::*,
+    AccountId, Error, RetryPolicy, SpacewalkRuntime, SpacewalkSigner, SubxtError,
 };
 
 use async_trait::async_trait;
 use futures::{stream::StreamExt, FutureExt, SinkExt};
 use std::{future::Future, ops::RangeInclusive, sync::Arc, time::Duration};
 use subxt::{
-	rpc::JsonValue, BasicError, Client as SubxtClient, ClientBuilder as SubxtClientBuilder, Event,
-	Metadata, RpcClient, SubstrateExtrinsicParams, TransactionEvents, TransactionProgress,
+    extrinsic::Signer, rpc::JsonValue, BasicError, Client as SubxtClient, ClientBuilder as SubxtClientBuilder, Event,
+    Metadata, PolkadotExtrinsicParams, RpcClient, TransactionEvents, TransactionProgress,
 };
 use tokio::{sync::RwLock, time::timeout};
 
 cfg_if::cfg_if! {
-	if #[cfg(feature = "standalone-metadata")] {
-		const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 1..=1;
-		const DEFAULT_SPEC_NAME: &str = "standalone";
-	} else if #[cfg(feature = "parachain-metadata")] {
-		const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 3..=3;
-		const DEFAULT_SPEC_NAME: &str = "parachain";
-		pub const SS58_PREFIX: u16 = 2032;
-	}
+    if #[cfg(feature = "standalone-metadata")] {
+        const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 1..=1;
+        const DEFAULT_SPEC_NAME: &str = "pendulum-parachain";
+    } else if #[cfg(feature = "parachain-metadata")] {
+        const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 3..=3;
+        const DEFAULT_SPEC_NAME: &str = "parachain";
+        pub const SS58_PREFIX: u16 = 2032;
+    }
 }
 const TRANSACTION_TIMEOUT: Duration = Duration::from_secs(300); // 5 minutes
 
-type RuntimeApi =
-	metadata::RuntimeApi<SpacewalkRuntime, SubstrateExtrinsicParams<SpacewalkRuntime>>;
+type RuntimeApi = metadata::RuntimeApi<SpacewalkRuntime, PolkadotExtrinsicParams<SpacewalkRuntime>>;
```

Comment:
Changing this from `SubstrateExtrinsicParam` to `PolkadotExtrinsicParam` will ensure that subxt uses an extrinsic format where tips are only value and not pairs of values and asset code.

Quite surprising considering these names.

Reply:
wow, curious and weird, nice catch